### ISSUE TITLE
Wordpress integration

### ISSUE THREAD

**Issue by codeagencybe** [uptrace/uptrace#111]
Hello

I'm new into APM in general, I have only tried New Relic for a short time and I wonder how does this compare to New Relic, and specific how does it work for eg Wordpress applications (and others)?
Does this work "out of the box" to collect metrics, slow queries, slow php functions etc... or does this require also custom/special development from the application side to have it working in Uptrace? 

I know New Relic requires us to install their plugin. Is there some equivalent for Uptrace? 
Or can we do something in the root application to make it work? 

Thanks in advance and very nice solution! 
Looking forward to try it in a few of our projects.

**Comment by vmihailenco** [comment]
Hello,

Uptrace uses OpenTelemetry to receive data from your application, but it looks like none have created an OpenTelemetry plugin for WordPress so far.

You still can use [OpenTelemetry PHP](https://github.com/open-telemetry/opentelemetry-php#getting-started) to create spans manually, but I guess this is not what you are looking for.

Overall it looks like only [Symfony](https://github.com/open-telemetry/opentelemetry-php-contrib/tree/main/src/Symfony) is supported so far. So if you have a Symfony project, I could help you get started with Uptrace and OpenTelemetry.

**Comment by vmihailenco** [comment]
FYI I've started working on uptrace-php and a basic example is already working - https://github.com/uptrace/uptrace-php/tree/master/example/basic

**Comment by codeagencybe** [comment]
Hello @vmihailenco 
Does make this compatible with Wordpress "ouf of the box"? Its a bit pity that there is nothing yet for the biggest CMS in the world... If any application would benefit from a tracing tool, it would be Wordpress.

**Comment by vmihailenco** [comment]
@codeagencybe nope, unfortunately Wordpress is still missing an OpenTelemetry plugin.

**Comment by juslintek** [comment]
https://github.com/opentelemetry-php/contrib-auto-wordpress
